Automattic/jetpack
# My Jetpack: Add close button navigation into `ConnectionScreen` component

Related with #22649 

#### Changes proposed in this Pull Request:

* Create `CloseLink` component.
* Add it to `ConnectionScreen` component.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Open `My Jetpack` page.
* Navigate to `#/connection`.
* You should see the `X` button.
* Clicks on it and go back to `my-jetpack`.

#### Demo

![image](https://user-images.githubusercontent.com/1663717/154728280-7d69ac44-29ec-47db-91fc-276e893c750e.png)
